velox/type/Type.cpp

```diff
@@ -330,13 +330,11 @@ bool RowType::operator==(const Type& other) const {
   return true;
 }
 
-std::string RowType::toString() const {
-  std::stringstream ss;
-  ss << (TypeTraits<TypeKind::ROW>::name) << "<";
+void RowType::printChildren(std::stringstream& ss) const {
   bool any = false;
   for (size_t i = 0; i < children_.size(); ++i) {
     if (any) {
-      ss << ',';
```

Comment:
@majetideepak Deepak, this change triggered a lot of internal test failures. Can we perhaps, add another parameter "delimited" with default value of "," and set it to ", " in PlanNode::toString()?